### ISSUE TITLE
FileLogObserver raises when it sees an event contains a failure that hates getTraceback()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7850]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)| @wsanchez reported|
|-|-|
|Trac ID|trac#7850|
|Type|enhancement|
|Created|2015-04-14 20:00:24Z|
|Branch|https://github.com/twisted/twisted/tree/log-file-traceback-7850|

`FileLogObserver` raises when it observes an event that contains a `log_failure` key with an object that raises if you call `getTraceback()` on it.

<details><summary>Searchable metadata</summary>

```
trac-id__7850 7850
type__enhancement enhancement
reporter__wsanchez wsanchez
priority__normal normal
milestone__New_Logging_System New Logging System
branch__branches_log_file_traceback_7850 branches/log-file-traceback-7850
branch_author__wsanchez wsanchez
status__closed closed
resolution__fixed fixed
component__logger logger
keywords__ 
time__1429041624779504 1429041624779504
changetime__1481165866975195 1481165866975195
version__None None
owner__wsanchez wsanchez

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)|@ralphm set owner to @wsanchez|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_5"></a>|@ralphm commented|
|-|-|

Good to go! Please merge with the logger branch.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_6"></a>|@wsanchez set status to `closed`|
|-|-|

(In [44432]) Merge branches/log-file-traceback-7850: Handle exceptions when rendering tracebacks in file observer.

Author: wsanchez
Reviewer: ralphm
Fixes: [#7850](https://github.com/twisted/twisted/issues/7850)
